NewForce-at-Mountwest/bangazon-api-bucatini
# Orders ticket

# Description

Orders ticket is complete. User is able to get all orders or one order, with customer information or product information embedded.


## Related Ticket(s)

Ticket #4

## Steps to Test
Run OrderTest.cs, check for passing result
Navigate to ~/order, look for results,
Navigate to ~/order?_include=customers; look for results,
Navigate to ~/order?_include=products; look for results,
Navigate to ~/order?_include=completed; look for results,
Navigate to ~/order/{int<3}_include=(customers, products); look for results,

# Checklist:
- [ Somewhat ] I have commented my code, particularly in hard-to-understand areas
- [ N ] I have made corresponding changes to the documentation
- [ Maybe? ] My changes generate no new errors
- [ Y ] I have added tests that prove my fix is effective or that my feature works
- [ Y ] New and existing unit tests pass locally with my changes
